Makefile

```diff
@@ -36,8 +36,20 @@ apply-with-kind: verify build-with-kind ## Deploy the kwok controller from the c
 		$(HELM_OPTS) \
 		--set controller.image.repository=$(IMG_REPOSITORY) \
 		--set controller.image.tag=$(IMG_TAG) \
+		--set serviceMonitor.enabled=true \
 		--set-string controller.env[0].name=ENABLE_PROFILING \
-		--set-string controller.env[0].value=true
+		--set-string controller.env[0].value=true 
+
+e2etests: ## Run the e2e suite against your local cluster
+	cd test && go test \
+		-count 1 \
+		-timeout 30m \
+		-v \
+		./suites/$(shell echo $(TEST_SUITE) | tr A-Z a-z)/... \
+		--ginkgo.focus="${FOCUS}" \
+		--ginkgo.timeout=30m \
+		--ginkgo.grace-period=30m \
```

Comment:
We may be able to drop the grace period if we don't expect timeouts here. [Grace period is just used by Ginkgo](https://onsi.github.io/ginkgo/#mental-model-the-life-cycle-of-interruptions-and-the-graceperiod-decorator) to give a bit of extra time after a timeout to allow things to cleanup and exit (both at the It block level and also in any BeforeEach/AfterEach blocks). This was increased to allow for more time to perform cleanup during scale testing, but may not be necessary for this testing here.

Reply:
I'll set it to 5m just to be sure.